src/generate.js

```diff
@@ -17,33 +17,56 @@ const header = [
   { id: "suspended", title: "Suspended" },
   { id: "revoked", title: "Revoked" },
   { id: "revokeReason", title: "Revoke Reason" },
+  { id: "phoneNumber", title: "Phone Number"},
   { id: "longText", title: "Long Text" }
 ]
 
-exports.execute = (output, { count, email, memberless }) => {
+const headerDataGenerators = {
+  "email": ({ memberless, firstName, lastName, email }) =>   memberless ? "" : forPercent(75, () => faker.internet.email(firstName, lastName, email)),
+  "firstName": ({ firstName }) => firstName,
+  "lastName": ({ lastName }) => lastName,
+  "city":(_) => faker.address.city(),
+  "state": (_) => faker.address.stateAbbr(),
+  "zipCode": ({}) => faker.address.zipCode(),
+  "licenseNumber":(_) => faker.random.uuid(),
+  "dateIssued": ({ dateIssued }) => dateIssued,
+  "expirationDate": ({ expirationDate }) => expirationDate,
+  "suspended": (_) => forPercent(20, true),
+  "revoked": (_) => false,
+  "revokeReason": (_) => faker.lorem.words(10),
+  "phoneNumber": (_) => faker.phone.phoneNumber("###-###-####"),
+  "longText": (_) => faker.lorem.words(55),
+}
+
+exports.execute = (output, { count, email, memberless, headers }) => {
   const records = [];
-  const writer = csv.createObjectCsvWriter({ path: path.resolve(output), header });
+  const headersEnabled = headers.toString().split(",")
+  const headersToUse = header.filter(h => {
+    for (let header of headersEnabled) {
+      if (header === h.id) {
+        return true
+      }
+    }
+    return false
+  })
+  const writer = csv.createObjectCsvWriter({ path: path.resolve(output), header: headersToUse });
 
   for (let i = 0; i < count; i++) {
-    const firstName = faker.name.firstName();
-    const lastName = faker.name.lastName();
-    const { dateIssued, expirationDate } = getIssuedAndExpiredDate();
-
-    records.push({
-      email: memberless ? "" : forPercent(75, () => faker.internet.email(firstName, lastName, email)),
-      firstName,
-      lastName,
-      city: faker.address.city(),
-      state: faker.address.stateAbbr(),
-      zipCode: faker.address.zipCode(),
-      licenseNumber: faker.random.uuid(),
-      dateIssued,
-      expirationDate,
-      suspended: forPercent(20, true),
-      revoked: false,
-      revokeReason: faker.lorem.words(10),
-      longText: firstName.repeat(20),
-    });
+    const record = {}
+    const firstName = faker.name.firstName()
+    const lastName = faker.name.lastName()
+    const { dateIssued, expirationDate } = getIssuedAndExpiredDate()
+    headersToUse.forEach(h => {
+      record[h.id] = headerDataGenerators[h.id]({
```

Comment:
if someone specifies a header that you don't have a generator for they will just get nil. Alternatively you could have a default generator.